dpbench/benchmarks/pairwise_distance/pairwise_distance_numba_np.py

```diff
@@ -0,0 +1,18 @@
+# Copyright 2022 Intel Corp.
+#
+# SPDX-License-Identifier: Apache-2.0
+
+import numba as nb
+import numpy as np
+
+
+@nb.njit(parallel=True, fastmath=True)
+def pairwise_distance(X1, X2, D):
```

Comment:
numba_np implementation failed validation. Following error showed: 
Intel MKL ERROR: Parameter 13 was incorrect on entry to DGEMM .
Is there a place will use MKL ?

Reply:
The latest pipeline is not showing MKL error, however pairwise distance nuba_np is still failing: 

================ Benchmark pairwise_distance ========================

================ implementation numba_np ========================
implementation: numba_np
error states: -4
error msg: Validation failed

Is validation failure expected?

Reply:
After a review on pipeline, it seems following mkl libs are making numba_np validation fail:

 **LD_PRELOAD="${CONDA_PREFIX}/lib/libmkl_intel_ilp64.so ${CONDA_PREFIX}/lib/libmkl_tbb_thread.so ${CONDA_PREFIX}/lib/libmkl_core.so"** python -c "import dpbench; dpbench.run_benchmarks()"

Some observations:
1. without LD_PRELOAD mkl libs, the numba_np will pass validation.
2. Nan is observed when LD_PRELOAD is set: for example: 
    here is an example when case size is set to 4:
numba_np [[1.16129922 0.96464119] [       nan        nan]]
numba: [[0.72923054 0.62394497] [0.51538559 0.28274484]]